bugsnag/src/test/java/com/bugsnag/BugsnagTest.java

```diff
@@ -164,8 +164,8 @@ public void close() {
     }
 
     @Test
-    public void testFilters() {
-        bugsnag.setFilters("testfilter1", "testfilter2");
+    public void testRedactedKeys() {
```

Comment:
We should keep the filters tests as well to ensure that doesn't break whilst we still have both.